src/apps/AdminApp/AdminRoot.js

```diff
@@ -996,15 +998,14 @@ class AdminRoot extends Component {
                   this.isAllowed("admin") ?
                       <Grid.Row>
                           <Grid.Column width={13}>
-                      <ChatBox user={user}
-                               rooms={rooms}
-                               selected_janus={current_janus}
-                               selected_room={current_room}
-                               selected_group={current_group}
-                               selected_user={feed_user}
-                               gateways={gateways}
-                               gdm={this.state.gdm}
-                               onChatRoomsInitialized={this.onChatRoomsInitialized}/>
+                          <ChatBox user={user}
+                                   rooms={rooms}
+                                   selected_janus={current_janus}
+                                   selected_room={current_room}
+                                   selected_group={current_group}
+                                   selected_user={feed_user}
+                                   gateways={gateways}
+                                   gdm={this.state.gdm} />
```

Comment:
please don't remove indication for chat room initialization. It should be reflected to the user.

Reply:
Added indicator for status in AdminRoot.